flight/Libraries/math/coordinate_conversions.c

```diff
@@ -221,13 +221,13 @@ uint8_t RotFrom2Vectors(const float v1b[3], const float v1e[3], const float v2b[
 
 	// The first rows of rot matrices chosen in direction of v1
 	mag = VectorMagnitude(v1b);
-	if (fabs(mag) < 1e-30)
+	if (fabsf(mag) < 1e-30f)
```

Comment:
The value in either case is originally a float so there is no gain of precision by promotion to double in fabsf-- just a bunch of 0 stuffing.

https://en.wikipedia.org/wiki/Single-precision_floating-point_format#Exponent_encoding

The smallest normal value in single precision floating point (and we shouldn't be using subnormals) is ~ 10^-38.